clients/java/src/main/java/io/zeebe/client/impl/worker/JobWorkerImpl.java

```diff
@@ -161,7 +161,9 @@ private void poll(final JobPoller jobPoller) {
     // to avoid race conditions that would let us exceed the buffer size
     final int actualRemainingJobs = remainingJobs.get();
     if (!shouldPoll(actualRemainingJobs)) {
+      LOG.debug("Expected to activate for jobs, but still enough remain. Reschedule poll.");
```

Comment:
This seems like trace level to me - any reason to put it as debug?

Reply:
You're right, it was for my own debugging. I'll lower the log level